app/src/state/editor/edit.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { paste } from './edit';
+import { emptyDocument, Paragraph } from '../../core/document';
+import { editorDefaults, EditorState } from './types';
+import { assertSome } from '../../util';
+
+const testContent: Paragraph[] = [
+  {
+    speaker: 'paragraph_01',
+    content: [
+      { type: 'artificial_silence', length: 1 },
+      { type: 'artificial_silence', length: 1 },
+      { type: 'artificial_silence', length: 1 },
+    ],
+  },
+  {
+    speaker: 'paragraph_02',
+    content: [
+      { type: 'artificial_silence', length: 1 },
+      { type: 'artificial_silence', length: 1 },
+      { type: 'artificial_silence', length: 1 },
+    ],
+  },
+];
+
+test('test paste merge', () => {
+  const state: EditorState = {
+    ...editorDefaults,
+    document: { ...emptyDocument, content: JSON.parse(JSON.stringify(testContent)) },
+    currentTimePlayer: 1.0,
```

Comment:
could i get the same test but with `currentTimePlayer: 3` please (to test that inserting at the end of a paragraph works as well)